### ISSUE TITLE
Unify upload and history views

### ISSUE THREAD

**Issue by jorinvo** [skddc/sharesome#41]
I think it would be pretty handy to just have one screen. 
It happened to me before that I dragged a file into the history view accidentally.
The upload area could be a small thing at the top and the area could expand if you drag a file onto the page.

**Comment by skddc** [comment]
I don't like the idea tbh. Loading images takes a long time, and I'm oftentimes on slow connections. Having to explicitly open history (which I need *much* less often than upload) means loading it every time.

**Comment by jorinvo** [comment]
Good point! 
Getting to the upload screen should be really quick since many times you just open the page the moment you want to upload an image.

Maybe we could still allow dropping files onto the history view?

**Comment by skddc** [comment]
> Maybe we could still allow dropping files onto the history view?

Hmm, yes. That might be possible somehow. Will require some creative hacking. :)

**Comment by jorinvo** [comment]
@skddc or we only support modern browsers with this and us a drop event instead of a file input (something like this http://stackoverflow.com/questions/10261989/html5-javascript-drag-and-drop-file-from-external-window-windows-explorer).

**Comment by skddc** [comment]
Yes, we can keep the file input on upload, though. It has some advantages (e.g. I'm usually not dropping but clicking anywhere in the window to open the file dialog).

**Comment by jorinvo** [comment]
Yes. Sounds good!
